lib/blockchain.go

```diff
@@ -2017,6 +2017,17 @@ func (bc *Blockchain) ProcessBlock(desoBlock *MsgDeSoBlock, verifySignatures boo
 		if err = bc.postgres.UpsertBlock(nodeToValidate); err != nil {
 			err = errors.Wrapf(err, "ProcessBlock: Problem saving block with StatusBlockStored")
 		}
+
+		// We're not storing blocks in postgres, so we should store the actual blocks in the badgerdb.
```

Comment:
Well it does look like we are storing blocks in the upsert above. is the issue that we're just not checking postgres in GetBlock?